Apps/Admin/Client/Pages/AnalyticsPage.razor

```diff
@@ -0,0 +1,32 @@
+@page "/analytics"
+@attribute [Authorize(Roles = Roles.Admin)]
+@using HealthGateway.Admin.Client.Store.Analytics
+@inherits Fluxor.Blazor.Web.Components.FluxorComponent
+@inject IJSRuntime JSRuntime
+
+<PageTitle>Analytics</PageTitle>
+<HgPageHeading>Analytics</HgPageHeading>
+
+<HgBannerFeedback Severity="Severity.Error" IsEnabled="HasError" TResetAction="AnalyticsActions.ResetAnalyticsStateAction">
+    @AnalyticsState.Value.RequestError?.Message
+</HgBannerFeedback>
+
+<MudGrid>
+    <MudItem xs="6" lg="3">
+          <HgButton TopMargin="Breakpoint.Always" Disabled="@AnalyticsState.Value.IsLoading" EndIcon="@Icons.Material.Filled.Download" FullWidth="@true" Variant="Variant.Filled" Color="Color.Primary" @onclick="@(() => GetProfilesData())">User Profiles</HgButton>
```

Comment:
Nit: event handlers can be simplified from `@onclick="@(() => GetProfilesData())"` to `@onclick="GetProfilesData"`.